.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -43,15 +49,16 @@ body:
         - Devopness web app
         - Documentation
         - SDK-JS
+        - UI - Design System/React components
```

In other part of the document you use spaces before/after forward slash, so let's keep the same style convention through the document:
```suggestion
        - UI - Design System / React components
```